app/src/main/java/org/simple/clinic/summary/PatientSummaryScreenController.kt

```diff
@@ -117,8 +117,14 @@ class PatientSummaryScreenController @Inject constructor(
     val phoneNumbers = patientUuid
         .flatMap { patientRepository.phoneNumber(it) }
 
-    return Observables.combineLatest(sharedPatients, addresses, phoneNumbers)
-        .map { (patient, address, phoneNumber) -> { ui: Ui -> ui.populatePatientProfile(patient, address, phoneNumber) } }
+    val bpPassport = patientUuid
+        .flatMapSingle { patientRepository.bpPassportForPatient(it) }
```

Comment:
If a sync happens in the background while this patient summary is open, and this sync saves a Bp Passport for the patient, this won't get updated.

The repository should return an `Observable` instead so that change notifications can be refreshed on the UI.